HelpSource/Classes/Char.schelp

```diff
@@ -108,3 +108,48 @@ code::
 	[i,a = i.asAscii,a.isAlphaNum,a.isPrint,a.isPunct,a.isControl].postln;
 });
 ::
+
+method::isLower
+
+returns:: true if the char is a lowercase letter.
+
+method::isUpper
+
+returns:: true if the char is a uppercase letter.
+
+method::isPathSeparator
+
+returns:: true if the char is the platform specific path separator.
+
+method::isVowel
+
+returns:: true if the char is a vowel.
+
+method::isControl
+
+returns:: true if the char is a control character.
+
+method::asAscii
+
+returns:: the object itself.
+
+method::asUnicode
+
+returns:: a single Unicode value. Ascii is a subset of Unicode.
```

Comment:
"ASCII" should be written in all-caps as it is elsewhere in the article

Reply:
I changed what you've proposed earlier: the object itself. (so no ASCII mention)